### PR TITLE
Update weekly meeting notes from Oct. 2017 - Jan. 2018

### PR DESCRIPTION
**Pull request by flarnie**
Still going~ 🚂

### CODE REVIEW

File: meta/meeting-notes/2017-01-05-weekly-meeting.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Draft.js Weekly 01/05/18
```
Comment (existentialism) [inline meta/meeting-notes/2017-01-05-weekly-meeting.md]: Nit: filename should be 2018-01-05 :)

**Comment by mxstbr** [comment]
screen shot 2018-02-01 at 15 14 29

:cry:

**Comment by flarnie** [comment]
Yea - I need to write a post with more details, but it's something we now believe is out of scope for Draft.js to support.

**Comment by mxstbr** [comment]
Will be eagerly awaiting said post!

**Pull request closed without merging**